### PR TITLE
Upgrade: add v1.1.1/v1.1.2 -> v1.1.3 path

### PR DESCRIPTION
**Pull request by bk201**
- Add 1.1.1/1.1.2 -> v1.1.3  path (basically the same as upgrade to v1.1.2)
- Remove EOL versions in newer releases.

Related issue: https://github.com/harvester/harvester/issues/4877
  - The doc is at https://65829eca977581008152dfa1--harvester-preview.netlify.app/v1.2/upgrade/v1-1-1-to-v1-1-3#2-an-upgrade-is-stuck-when-pre-draining-a-node-case-2

### CODE REVIEW

**Review by jillian-maroket — COMMENTED** [review]
@bk201 Let me know if you have concerns about the changes.

File: versioned_docs/version-v1.1/upgrade/v1-1-1-to-v1-1-3.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+
+---
+
+### 2. An upgrade is stuck when pre-draining a node (case 2)
+
+An upgrade is stuck, as shown in the screenshot below:
+
+![](/img/v1.2/upgrade/known_issues/3216-stuck-pre-drain.png)
+
+And you can also observe that multiple nodes' status is `SchedulingDisabled`.
```
[the remark this answers is not in the document]
Comment (bk201) [reply versioned_docs/version-v1.1/upgrade/v1-1-1-to-v1-1-3.md:39]: I'll modify two at least "two" nodes here.

**Comment by bk201** [comment]
Accept suggested changes and fix all versions.

File: docs/upgrade/v1-1-1-to-v1-1-3.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+
+---
+
+### 1. The upgrade process is stuck when pre-draining a node. (Case 1)
+
+An upgrade is stuck, as shown in the screenshot below:
+
+![](/img/v1.2/upgrade/known_issues/3216-stuck-pre-drain.png)
+
+Harvester is unable to proceed with the upgrade and the status of at least two node are `SchedulingDisabled`.
```
Comment (jillian-maroket) [inline docs/upgrade/v1-1-1-to-v1-1-3.md:37]:
```suggestion
Harvester is unable to proceed with the upgrade and the status of two or more nodes is `SchedulingDisabled`.
```

**Review by jillian-maroket — COMMENTED** [review]
@bk201 I noticed 2 errors in the updated text. Please check.

**Review by jillian-maroket — APPROVED** [review]
@bk201 LGTM. Thanks!

**Review by markhillgit — APPROVED** [review]
looks good thanks

**Pull request merged**